test/src/main/java/org/axonframework/test/aggregate/FixtureConfiguration.java

```diff
@@ -105,6 +105,14 @@
      */
     FixtureConfiguration<T> withSubtypes(Class<? extends T>... subtypes);
 
+    /**
+     * Configures the fixture for state stored aggregates.
```

Comment:
A small reference towards the `givenState` method in this JavaDoc would be good, as that can better guide readers of the documentation on what to do next after using this method.